06-express-and-layered-architecture/src/cart/order/order.service.ts

```diff
@@ -0,0 +1,26 @@
+import { createOrder as createOrderRepository } from "./order.repository";
+import { deleteCart as deleteCartRepository } from "../cart.repository";
+import { getCart as getCartRepository } from "../cart.repository";
+
+export async function createOrder(
```

Comment:
The function is called createOrder, but it does three things: gets the cart, creates the order and then deletes the cart. Think about the possibilities of using middleware for each of these functionalities: one middleware for getting the cart, on fro creating the order and finally one for deleting the contents of the cart. Would that make sense?